libbeat/docs/communitydashboards.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[community-beat-dashboards]]
+== Community Beat dashboards
+
+The open source community has been hard at work developing new Beats. You can check
```

Comment:
No, there is none for now. I know it's a bit strange. Maybe we can add ... be the first to contribute ...